darwin/Platform.c

```diff
@@ -84,6 +72,17 @@ in the source distribution for its full text.
 #include <mach/mach_time.h>
 #endif
 
+#ifndef HAVE_NET_IF_TYPES_H
+#ifndef IFT_LOOP
+#define IFT_LOOP 24 /* Software loopback network */
+#endif
+#endif
+
+#ifndef HAVE_NET_ROUTE_H
+#ifndef RTM_IFINFO2
+#define RTM_IFINFO2 0x12
+#endif
+#endif
```

Comment:
Need explanation on why we define the macro constants to the values presented. If a system doesn't have the macro constants defined, it is likely that the properties associated to the constants are not applicable to the system, in which case we should compile out the relevant code rather than define the constants to the value that you (@tslmy) even don't know what they do.

Reply:
Deleted. No longer applies. I'll resolve this conversation.